### DISCUSSION TITLE
nativephp and alpine js not function

### DISCUSSION THREAD

**Discussion by LuisGB2212** [NativePHP/laravel#330] [General]
NativePHP does not work for me with alpinejs functions, the dropdowns are with x.data to open them and when loading the dropdowns are visible.

**Comment by simonhamp** [comment]
@LuisGB2212 this requires a lot more detail to enable folks to help.

**Reply by discoverlance-com** [reply]
It's also not working for me here too. I am using filamentphp and I decided to ship it with native php to experiment and some of the issues in my console are:

```js
support.js?v=3.2.114.0:1 Uncaught 
TypeError: Cannot read properties of undefined (reading 'from')
    at J (support.js?v=3.2.114.0:1:2141)
    at ne (support.js?v=3.2.114.0:1:1949)
    at support.js?v=3.2.114.0:1:10728
    at support.js?v=3.2.114.0:1:10821
    at support.js?v=3.2.114.0:1:216
    at support.js?v=3.2.114.0:1:38997
    at support.js?v=3.2.114.0:25:12146


livewire.js?id=cc800bf4:1125 Alpine Expression Error: window.pluralize is not a function

Expression: "
                window.pluralize('1 record selected|:count records selected', selectedRecords.length, {
                    count: selectedRecords.length,
                })
            "
```

By default, all my dropdowns are automatically open, I am not able to close them and when I click on the dropdown I also see this errors in my console.


```js
Uncaught TypeError: $refs.panel.toggle is not a function
    at Proxy.toggle (eval at safeAsyncFunction (livewire.js?id=cc800bf4:1176:21), <anonymous>:5:25)
    at runIfTypeOfFunction (livewire.js?id=cc800bf4:1212:26)
    at livewire.js?id=cc800bf4:1200:11
    at tryCatch (livewire.js?id=cc800bf4:1118:14)
    at livewire.js?id=cc800bf4:3862:7
    at handler4 (livewire.js?id=cc800bf4:3166:27)
    at livewire.js?id=cc800bf4:3230:9
    at HTMLDivElement.<anonymous> (livewire.js?id=cc800bf4:3168:54)

```

This is a picture of the dropdowns open by default.

![image](https://github.com/user-attachments/assets/a4178bb1-f204-460e-a672-b67bfe7624ff)


Electron does show a warning about csp in the app console but I am not sure if it contributes to the issue

```js
Electron Security Warning (Insecure Content-Security-Policy) This renderer process has either no Content Security
  Policy set or a policy with "unsafe-eval" enabled. This exposes users of
  this app to unnecessary security risks.

```

**Reply by discoverlance-com** [reply]
I think for what I am experiencing in particular with FilamentPHP, I have found this discussion, [Filament V3 with Native PHP: issue with userMenuItems dropdown menu](https://github.com/filamentphp/filament/discussions/12388) to be relevant to it so I am following the suggestions there. I am not sure though how it directly affects the alpinejs question here.

Thanks.

**Reply by simonhamp** [reply]
Hard to say, but feels unrelated
